### PR TITLE
LOG-1279: Add placeholder for linting tasks

### PR DESCRIPTION
**Pull request by jcantrill**
### Description
This PR adds a placeholder  to be executed as a CI job for linting

### Links
https://issues.redhat.com/browse/LOG-1279

cc @ewolinetz @periklis

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -106,3 +106,7 @@ run: copy-es-certs
 		--cl-infra-role-name=sg_role_admin \
 		--ssl-insecure-skip-verify
 .PHONY: run
+
+run-linter:
```
Comment (alanconway) [inline Makefile:110]: Suggest target name "lint:", same as CLO. Your call.
/lgtm
/hold

Comment (jcantrill) [reply Makefile]: updated to "lint"

**Pull request merged**